README.md

```diff
@@ -14,107 +14,87 @@ Assumptions:
 
 ### Install Prerequisites and Code
 
-1. If you're not already running Node.js, download and run the installer for your operating system. We recommend v4.x: [https://nodejs.org/en/](https://nodejs.org/en/ "Node.js installer").
+1. If you're not already running Node.js, download and run the installer for your operating system. We recommend v10.x: [https://nodejs.org/en/](https://nodejs.org/en/ "Node.js installer").
 
-2. Use *npm* (Node's package manager, which is part of the Node.js install) to install the latest version of gulp. This is necessary for runing the babel version of the `gulpfile`):
-
-    ```bash
-        $ npm install gulp && npm install gulp -g
-    ```
-
-3. From the command line, clone the DATA Act web app repository from GitHub to your local machine:
+2. From the command line, clone the DATA Act web app repository from GitHub to your local machine:
 
         $ git clone https://github.com/fedspendingtransparency/data-act-broker-web-app.git
 
-4. Switch to the alpha release version of the code. This is the latest stable release.
+3. Switch to the alpha release version of the code. This is the latest stable release.
```

Comment:
No longer applies / this step can be removed. We could keep a note about `master` being the latest stable release and `staging` as the work-in-progress version.

Reply:
👌